backend-project/small_eod/generic/tests.py

```diff
@@ -31,11 +32,22 @@ def get_url(self, name, **kwargs):
             raise NotImplementedError("get_url must be overridden or basename defined")
         return reverse("{}-{}".format(self.basename, name), kwargs=kwargs)
 
-    def test_list_plain(self):
+    def test_dict_plain(self):
+        parsed_response_len = 4
         response = self.client.get(self.get_url(name="list", **self.get_extra_kwargs()))
+        parsed_response = response.json()
+        response_result = parsed_response.get(self.response_results_key)
+
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.json()), 1)
-        self.validate_item(response.json()[0])
+        self.assertIsNotNone(response_result)
+        self.assertEqual(len(parsed_response), parsed_response_len)
+        self.validate_item(response_result[0])
+
+    # def test_list_plain(self):
```

Comment:
To przed wcieleniem będzie należało usunąć. Ale myślę, że spojrzenie na to dobrze pokazuje podobieństwo.